src/Malawi_cor_labels_GEE script.js

```diff
@@ -0,0 +1,342 @@
+         [33.98742088899353, -12.254869172764044],
+         [33.953431936356814, -12.262921037431857],
+         [33.95806679353455, -12.220142692759165]]),
+    Sentinel2 = ee.Image("projects/ee-aasareansah/assets/Sent_Sept_Comp_Malawi"),
+    Malawi_EPA = ee.FeatureCollection("projects/ee-aasareansah/assets/NASA_Harvest/Malawi_EPAs"),
+    Malawi_TA = ee.FeatureCollection("projects/ee-aasareansah/assets/NASA_Harvest/Malawi_TAs"),
+    Admin = ee.FeatureCollection("projects/ee-aasareansah/assets/NASA_Harvest/Malawi_admin2");
+
+
+//Rename b1 as True value 
+var upd_name = malawi_binary.select('b1').rename('Wrong value');
+
+var leftMap = ui.Map();
+var layer1 = ui.Map.GeometryLayer([], 'Geometry Layer', 'red');
+var drawingTools1 = leftMap.drawingTools();
+drawingTools1.layers().add(layer1);
+
+var rightMap = ui.Map();
+var layer2 = ui.Map.Layer(ee.FeatureCollection([]), {'color': 'red'}, 'Points');
+
+// Link two layers together
+drawingTools1.onDraw(function(geometry, layer) {
+  if (layer === layer1) {
+    layer2.setEeObject(layer.getEeObject());
+  }
+});
+drawingTools1.onEdit(ui.util.debounce(function(geometry, layer) {
+  if (layer === layer1) {
+    layer2.setEeObject(layer.getEeObject());
+  }
+}, 100));
+drawingTools1.onErase(function(geometry, layer) {
+  if (layer === layer1) {
+    layer2.setEeObject(layer.getEeObject());
+  }
+});
+
+//Set map center using this long lat and zoom level
+rightMap.setCenter(33.78, -12.27, 11);
+
+// Make the default basemap the satellite layer
+rightMap.setOptions('satellite');
+
+//Specify the point as the only drawing tool 
+drawingTools1.setDrawModes(['point']);
+
+//Assign specific palettes to the cropland map
+var palettes = require('users/gena/packages:palettes');
+var palette = palettes.cmocean.Speed[7];
+
+// Use this empty image for paint().
+var empty = ee.Image().byte();
+
+var Admin_color = empty.paint({
+featureCollection: Admin,
+color: 1,
+width: 2
+});
+
+var EPA_color = empty.paint({
+featureCollection: Malawi_EPA,
+color: 1,
+width: 2
+});
+
+var TA_color = empty.paint({
+featureCollection: Malawi_TA,
+color: 1,
+width: 2
+});
+
+//Display the cropland map on the left panel
+var vis_crop = {min: 0, max: 1.0, palette: palette.slice(0,-2)};
+leftMap.addLayer(malawi_binary, vis_crop, 'Malawi cropland');
+leftMap.addLayer(Admin_color, {palette: 'CE33FF'}, 'Malawi_Admin');
+leftMap.addLayer(EPA_color, {palette: '0096FF'}, 'Malawi_EPA', false);
+leftMap.addLayer(TA_color, {palette: 'black'}, 'Malawi_TA', false);
+
+//Visualize the Sentinel 2 high resolution image on the right panel
+var vis = {min: 0.0, max: 0.3, bands: ['B4', 'B3', 'B2']};
+rightMap.addLayer(Sentinel2, vis, 'Sentinel-2');
+rightMap.add(layer2);
+rightMap.addLayer(Admin_color, {palette: 'CE33FF'}, 'Malawi_Admin');
+rightMap.addLayer(EPA_color, {palette: '0096FF'}, 'Malawi_EPA', false);
+rightMap.addLayer(TA_color, {palette: 'black'}, 'Malawi_TA', false);
+
+// Link the two panels
```

Comment:
> we should add comments only in location where it is necessary and code itself cannot express what is happening there.

https://www.todaysoftmag.com/article/1120/clean-code-comments-and-formatting

Generally I recommend removing lot's of these comments, the code should make it clear what is happening on its own.